www/src/templates/template-docs-markdown.js

```diff
@@ -15,6 +15,29 @@ import Breadcrumb from "../components/docs-breadcrumb"
 import Container from "../components/container"
 import PrevAndNext from "../components/prev-and-next"
 
+import { useActiveHash } from "../hooks/use-active-hash"
+
+const getHeadingIds = (toc, traverseFullDepth = false) => {
```

Comment:
I think it makes more sense to pull this function into the table of contents component. For example, if we were to implement #21808, we'd have to repeat the logic right now.

Reply:
@tesseralis Done in e567fe2